### PR TITLE
Updated snipping documentation + clean up and refactor move_to_map_pokemon.py

### PR DESCRIPTION
**Pull request by YvesHenri**
Updated config docs: deprecated "map_path" and "min_time" (snipping).

### CODE REVIEW

**Comment by YvesHenri** [comment]
Fixing some issue.

**Comment by YvesHenri** [comment]
Seems to have been fixed. Needs testing.

File: pokemongo_bot/cell_workers/move_to_map_pokemon.py [hunk 1]
```diff
@@ -435,15 +422,6 @@ def _move_to(self, pokemon):
         return walker_factory(self.walker, self.bot, pokemon['latitude'], pokemon['longitude'])
 
     def _move_to_pokemon_througt_fort(self, fort, pokemon):
-        """Moves trainer towards a fort before a Pokemon.
```
Comment (solderzzc) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: why remove the comment?

**Comment by alexyaoyang** [comment]
I changed the PR title for you, since you do have quite some changes in `move_to_map_pokemon.py`, it should reflect on the title :)

File: pokemongo_bot/cell_workers/move_to_map_pokemon.py (hunk 1 above)
Comment (YvesHenri) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: Function name is straight forward. No need documentation (cleanup).

**Comment by YvesHenri** [comment]
@alexyaoyang Those are very minor refactors, so... Anyways, thank you.

@everyone I'm extremely busy these past days. I might have spot a bug where it intends to snipe the same target (is_inspected related), due to recent refactors. I'll be thankful if someone could test it.

**Comment by YvesHenri** [comment]
New fully refactored sniper version coming soon (lots of updates).

**Pull request closed without merging**